pkg/webhook/v1alpha1/placemanifestwork_validating.go

```diff
@@ -55,6 +55,12 @@ func (r *PlaceManifestWorkWebhook) validateRequest(newPlaceWork *workv1alpha1.Pl
 }
 
 func validatePlaceManifests(placeWork *workv1alpha1.PlaceManifestWork) error {
+	if placeWork.Spec.PlacementRef.Name == "" {
+		return apierrors.NewBadRequest("Placement name should not be empty")
+	}
 
+	if len(placeWork.Spec.ManifestWorkTemplate.Workload.Manifests) == 0 {
+		return apierrors.NewBadRequest("ManifestWorkTemplate workload should not be empty")
+	}
```

Comment:
you may also want to validate the payload in manifests by reusing the same func for manifestwork validation